src/xdocs/config_imports.xml

```diff
@@ -1159,6 +1159,10 @@ public class InputEclipseStaticImportsOrder { }
       <subsection name="Error Messages">
         <ul>
           <li>
+            <a href="https://github.com/search?q=path%3Asrc%2Fmain%2Fresources%2Fcom%2Fpuppycrawl%2Ftools%2Fcheckstyle%2Fchecks%2Fimports+filename%3Amessages*.properties+repo%3Acheckstyle%2Fcheckstyle+%22import.groups.separated.internally%22">
+            import.groups.separated.internally</a>
+          </li>
+          <li>
```

Comment:
I update config_imports.xml, but i don't assured that it's all that need. For new functionality don't need add additional options to examples config.